UIComponents/src/main/java/com/braintreepayments/api/uicomponents/PayPalButton.kt

```diff
@@ -26,6 +39,17 @@ class PayPalButton @JvmOverloads constructor(
     private val desiredHeight = resources.getDimension(R.dimen.pay_button_height).toInt()
     private val minDesiredWidth = resources.getDimension(R.dimen.pay_button_min_width).toInt()
 
+    private var payPalRequest: PayPalRequest? = null
+
+    /**
+     * The PayPal client used to create payment auth requests and tokenize results.
+     * Must be initialized via [initialize] before use.
+     */
+    lateinit var payPalClient: PayPalClient
+    private val payPalLauncher: PayPalLauncher
+
+    var payPalLaunchCallback: PayPalLaunchCallback? = null
```

Comment:
Can we add a kdoc for this variable? It would be good to give merchants some context on what this callback is for.

There might be a case where the merchant doesn't set this callback but the button is clicked by the buyer. What are your thoughts on adding a check to see if the `payPalLaunchCallback` is null before executing anything in the onClickListener? I feel that failing faster is better to be able to notify the merchant that they have not integrated the button correctly.